### ISSUE TITLE
REPL user share commands crash the prompt

### ISSUE THREAD

**Issue by crtahlin** [jmozah/intOS-dfs#75]
inbox variant
```  
dfs@demo8 >>> test1 >> / > user share inbox
panic: runtime error: index out of range [3] with length 3

goroutine 1 [running]:
github.com/jmozah/intOS-dfs/cmd/dfs/cmd.executor(0xc0003d24b0, 0x10)
	/home/crtah/Documents/Swarm/intOS-dfs/cmd/dfs/cmd/prompt.go:275 +0x5a71
github.com/c-bata/go-prompt.(*Prompt).Run(0xc0000c2000)
	/home/crtah/Documents/Swarm/intOS-dfs/vendor/github.com/c-bata/go-prompt/prompt.go:85 +0x838
github.com/jmozah/intOS-dfs/cmd/dfs/cmd.initPrompt()
	/home/crtah/Documents/Swarm/intOS-dfs/cmd/dfs/cmd/prompt.go:83 +0x10b
github.com/jmozah/intOS-dfs/cmd/dfs/cmd.glob..func1(0x106d700, 0x10e9ef8, 0x0, 0x0)
	/home/crtah/Documents/Swarm/intOS-dfs/cmd/dfs/cmd/prompt.go:66 +0xfa
github.com/spf13/cobra.(*Command).execute(0x106d700, 0x10e9ef8, 0x0, 0x0, 0x106d700, 0x10e9ef8)
	/home/crtah/Documents/Swarm/intOS-dfs/vendor/github.com/spf13/cobra/command.go:846 +0x2c2
github.com/spf13/cobra.(*Command).ExecuteC(0x106d9a0, 0xc000010018, 0xc0002c5f58, 0x1)
	/home/crtah/Documents/Swarm/intOS-dfs/vendor/github.com/spf13/cobra/command.go:950 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
	/home/crtah/Documents/Swarm/intOS-dfs/vendor/github.com/spf13/cobra/command.go:887
github.com/jmozah/intOS-dfs/cmd/dfs/cmd.Execute()
	/home/crtah/Documents/Swarm/intOS-dfs/cmd/dfs/cmd/root.go:65 +0xa5
main.main()
	/home/crtah/Documents/Swarm/intOS-dfs/cmd/dfs/main.go:24 +0x25

``` 

outbox variant 

``` 
dfs@demo8 >>>user share outbox
panic: runtime error: index out of range [3] with length 3

goroutine 1 [running]:
github.com/jmozah/intOS-dfs/cmd/dfs/cmd.executor(0xc000286fe0, 0x11)
	/home/crtah/Documents/Swarm/intOS-dfs/cmd/dfs/cmd/prompt.go:275 +0x5a71
github.com/c-bata/go-prompt.(*Prompt).Run(0xc000332000)
	/home/crtah/Documents/Swarm/intOS-dfs/vendor/github.com/c-bata/go-prompt/prompt.go:85 +0x838
github.com/jmozah/intOS-dfs/cmd/dfs/cmd.initPrompt()
	/home/crtah/Documents/Swarm/intOS-dfs/cmd/dfs/cmd/prompt.go:83 +0x10b
github.com/jmozah/intOS-dfs/cmd/dfs/cmd.glob..func1(0x106d700, 0x10e9ef8, 0x0, 0x0)
	/home/crtah/Documents/Swarm/intOS-dfs/cmd/dfs/cmd/prompt.go:66 +0xfa
github.com/spf13/cobra.(*Command).execute(0x106d700, 0x10e9ef8, 0x0, 0x0, 0x106d700, 0x10e9ef8)
	/home/crtah/Documents/Swarm/intOS-dfs/vendor/github.com/spf13/cobra/command.go:846 +0x2c2
github.com/spf13/cobra.(*Command).ExecuteC(0x106d9a0, 0xc000010018, 0xc000245f58, 0x1)
	/home/crtah/Documents/Swarm/intOS-dfs/vendor/github.com/spf13/cobra/command.go:950 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
	/home/crtah/Documents/Swarm/intOS-dfs/vendor/github.com/spf13/cobra/command.go:887
github.com/jmozah/intOS-dfs/cmd/dfs/cmd.Execute()
	/home/crtah/Documents/Swarm/intOS-dfs/cmd/dfs/cmd/root.go:65 +0xa5
main.main()
	/home/crtah/Documents/Swarm/intOS-dfs/cmd/dfs/main.go:24 +0x25
crtah@pop-os:~/Documents/Swarm/intOS-dfs$ 



```

**Comment by jmozah** [comment]
fixed in comit 7d49bc3decb12f5fbbdb1b79b220d52e356f03ab

**Comment by crtahlin** [comment]
verified, doesnt crash
